src/Visualization.jl

```diff
@@ -9,17 +9,17 @@ const MAX_EDGE_WIDTH = 10.
 const MAX_ARROW_SIZE = 25.
 const MAX_NODE_SIZE = 40.
 
-function Makie.plot(path::ContractionPath; colormap = to_colormap(:viridis)[begin:end-10], kwargs...)
+function Makie.plot(path::ContractionPath; colormap = to_colormap(:viridis)[begin:end-10], labels = false, kwargs...)
```

Comment:
```suggestion
function Makie.plot(path::ContractionPath; kwargs...)
```

Reply:
> I like it!
> 
> 1. You don't need to repeat the keyword arguments if you're not gonna use it directly in the `plot` function. Just pass them to `plot!` as `kwargs`.
> 2. Where do you use `elabels`? I think you didn't commit that part.

Thanks!
1. I was keeping the keyword arguments since we might want to use `plot` directly if we create a canvas and we have multiple subplots in it, which I think it makes sense.
2. Elabels are passed as an argument inside the `graphplot!()` function with: `elabels = labels ? elabels : nothing`.

Reply:
It's still redundant because you can still pass this keyword arguments without explicitly defining them in `plot`. I would quit them.